SDK/AppCenterData/Microsoft.AppCenter.Data.iOS/PaginatedDocuments.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Microsoft.AppCenter.Data.iOS.Bindings;
+
+namespace Microsoft.AppCenter.Data
+{
+    public partial class PaginatedDocuments<T>
+    {
+        private MSPaginatedDocuments PaginatedDocumentsInternal { get; }
```

Comment:
```suggestion
        private MSPaginatedDocuments PaginatedDocuments { get; }
```

Reply:
I made this changes but it won't actually work because it's the same as the class name